app/src/main/res/values/strings.xml

```diff
@@ -194,102 +194,102 @@
     <string name="ui_theme">UI theme (app restart required)</string>
     <string name="theme_light">Light</string>
     <string name="theme_dark">Dark</string>
-    <string name="dont_show_boost">Don\'t show boost</string>
-    <string name="dont_show_reply">Don\'t show reply</string>
+    <string name="dont_show_boost">Don\'t show boosts</string>
+    <string name="dont_show_reply">Don\'t show replies</string>
     <string name="regex_filter">Regex filter</string>
     <string name="regex_error">Regex error</string>
     <string name="notification_delete">Empty notifications</string>
-    <string name="confirm_delete_notification">This will empty your notification on your instance.\nAre you sure?</string>
+    <string name="confirm_delete_notification">Empty all notifications on instance?</string>
     <string name="search_for">Search %1$s</string>
-    <string name="spoil_visibility_for_account">Visibility is spoiled for account.</string>
+    <string name="spoil_visibility_for_account">The account is visible.</string>
```

Comment:
no, this message is used when switching account in post composing activity. if switched account has more smaller visibility, visibility selection is changed to  spoiled.

Reply:
I don't understand this. Could possibly be "spoilt".
What happens exactly?